docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -75,14 +114,14 @@ D. Does not attend ANC at all during pregnancy
     - C
     - D
 
-The below table describes what propensity values to use for each exposure option outlined above.
+The below table describes what probability values to use for each exposure option outlined above.
```

Comment:
```suggestion
The below table describes what probability values to use for each exposure option outlined above, **for full term pregnancies**.
```
Moving the restriction to full term pregnancies from above, as noted in [my other comment](https://github.com/ihmeuw/vivarium_research/pull/1695/files#r2170182924).